packages/mercury-case-analysis/src/rating.ts

```diff
@@ -0,0 +1,163 @@
+import { generateObject, LanguageModel } from "mongodb-rag-core/aiSdk";
+import { extractZodSchemaDescriptions } from "./utils";
+import z from "zod";
+import { stripIndent } from "common-tags";
+import YAML from "yaml";
+
+const ratingSchema = z
+  .number()
+  .min(1)
+  .max(5)
+  .describe("A score from 1 (poor) to 5 (excellent).");
```

Comment:
imo somehwere in this file should be description of what 1-5 mean. maybe something like likert score etc